docs/source/notes/extending.rst

```diff
@@ -605,14 +607,14 @@ provides a developer-facing API for ensuring full support for
 changes without warning in the future.
 
 First, to get a listing of all overridable functions, use
-``torch.overrides.get_overridable_functions``. This returns a dictionary whose
+``torch.overrides._get_overridable_functions``. This returns a dictionary whose
```

Comment:
Ooh... I forgot that you refer to this explicitly in the docs. Referencing a private function here is kind of weird. Perhaps that means this particular function should be kept public. `_get_testing_overrides` and `_get_ignored_functions` are the ones that definitely should not be public.

Reply:
Version 1.7 is the only version where these were public. Before that, the submodule was private. v1.7.1 is the only version where the docs might show it as public.

Reply:
Sorry, it doesn't have to do with the historical context is. What I mean is, it's weird to mark a function with a leading underscore, and then immediately turn around and in the docs say, "Hey, you should call this private function to find out what functions to implement it with!" It's like saying, "This thing, which you're not supposed to use, is so useful that we are going to tell you about it in the public documentation hint hint wink wink"

Reply:
I'll make these public.